oxapy/__init__.py

```
@@ -8,7 +11,143 @@
 import hashlib
 
 from functools import partial
-from .oxapy import *  # ty:ignore[unresolved-import]
+
+from watchdog.observers import Observer
+from watchdog.events import PatternMatchingEventHandler
+
+from .oxapy import *
```

Ruff (0.15.21)

[error] `from .oxapy import *` used; unable to detect undefined names

(F403)